.ci/Jenkinsfile

```diff
@@ -71,11 +71,28 @@ pipeline {
      Execute unit tests.
      */
     stage('Test') {
-      steps {
-        cleanup()
-        withMageEnv(){
-          dir("${BASE_DIR}"){
-            sh(label: 'Runs the (unit) tests',script: 'mage -debug test|tee tests-report.txt')
+      parallel {
+        stage('Test on Linux') {
+          options { skipDefaultCheckout() }
+          steps {
+            cleanup()
+            withMageEnv(){
+              dir("${BASE_DIR}"){
+                sh(label: 'Runs the (unit) tests',script: 'mage -debug test|tee tests-report.txt')
+              }
+            }
+          }
+        }
+        stage('Test on Windows') {
+          agent { label 'windows-10' }
+          options { skipDefaultCheckout() }
+          steps {
+            cleanup()
+            withMageEnv(){
+              dir("${BASE_DIR}"){
+                sh(label: 'Runs the (unit) tests',script: 'mage -debug test|tee tests-report.txt')
```

Comment:
As this is run in Windows, it should be called `bat`
it also seems it can be called `cmd` so it uses the right call under the hood.
https://github.com/elastic/apm-pipeline-library/blob/main/vars/cmd.groovy